scripts/apps/authoring/authoring/directives/AuthoringDirective.ts

```diff
@@ -1032,13 +1032,26 @@ export function AuthoringDirective(
                     }
                 }
 
-                $rootScope.$broadcast('item:nextStage', {
-                    stage: stageList[stageIndex],
-                    itemId: $scope.item._id,
-                    selectedStage: selectedStage,
-                    selectedDesk: selectedDesk,
-                    item: $scope.item,
-                });
+                if ($location.$$path === '/workspace') {
+                    api.save('move', {}, {
+                        task: {desk: selectedDesk._id, stage: stageList[stageIndex]._id},
```

Comment:
I'm also concerned about duplication of code. Would it be better if we had this either in the SuperdeskApi, or in the internal api (https://github.com/superdesk/superdesk-client-core/blob/develop/scripts/api/article.ts).

This way the actual sending of content (api calls) could be managed in the one place